src/app/psat/psatConstants.ts

```diff
@@ -121,4 +121,57 @@ export const motorEfficiencyConstants: Array<{ value: number, display: string }>
         value: 3,
         display: 'Specified'
     }
-]
\ No newline at end of file
+]
+
+// Ordered by pumpTypeConstants
+export const pumpTypeRanges: Array<{ range: number[], value: number }> = [
+    {
+        value: 0,
+        range: [100, 20000]
```

Comment:
Instead of having range: Array<number> and then accessing min/max them by index (may not be intuitive to the next guy)..

make `range: {min: number, max: number}`